recipes/easyexif/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+import os
+from conans import CMake, ConanFile, tools
+
+class EasyExifConan(ConanFile):
+    name = "easyexif"
+    description = "Tiny ISO-compliant C++ EXIF parsing library, third-party dependency free."
+    topics = ("conan", "exif", "image", "multimedia", "format", "graphics")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mayanklahiri/easyexif"
+    license = "https://github.com/mayanklahiri/easyexif/blob/master/LICENSE"
```

Comment:
Isn't it `MIT`?
https://mit-license.org/

Reply:
I tried to match it against MIT or BSD, and it slightly differs. Do not know if it was intended by the author but I think it is better to set an explicit path to license.

Reply:
It's BSD-2-Clause https://spdx.org/licenses/BSD-2-Clause.html